content/events/filecoin-orbit-showcase-hr.md

```diff
@@ -0,0 +1,22 @@
+---
+eventName: "Filecoin Orbit Ecosystem Showcase & Happy Hour"
+eventDescription: "Taking place around ETH Denver, the Filecoin Orbit community event provides an opportunity to immerse yourself in the Filecoin ecosystem, learn from the brightest minds in the industry, and connect with like-minded community members. Join us at The Shop in CSU Spur Denver!" 
+
+eventTopic: "Filecoin"
+eventType: "Showcase" "Happy Hour"
```

Comment:
@laeccles The eventType can have one among the following values: Conference, Virtual, Social. Please select one from the above.